### PR TITLE
[16.0][ADD] stock_release_channel_plan_process_end_time

### PR DESCRIPTION
**Pull request by jbaudoux**
Glue module between:

- https://github.com/OCA/wms/pull/550
- https://github.com/OCA/wms/pull/695

cc @lmignon @sbejaoui @rousseldenis

### CODE REVIEW

**Review by lmignon — CHANGES_REQUESTED** [review]
Can you add in a specific commit a test-requirements.txt file with the dependent PR as it's done here https://github.com/OCA/wms/pull/693/commits/758b160304e097300932c358c7dd8bc915db0c45

**Comment by jbaudoux** [comment]
> Can you add in a specific commit a test-requirements.txt file with the dependent PR as it's done here [758b160](https://github.com/OCA/wms/commit/758b160304e097300932c358c7dd8bc915db0c45)

Done. The failing test is not on this PR but on https://github.com/OCA/wms/pull/550

File: stock_release_channel_plan_process_end_time/tests/test_stock_release_channel_plan.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+from odoo import fields
+
+from odoo.addons.partner_tz.tools.tz_utils import tz_to_utc_naive_datetime
+from odoo.addons.stock_release_channel_plan.tests import test_stock_release_channel_plan
+from odoo.addons.stock_release_channel_process_end_time.utils import float_to_time
+
+
+class TestStockReleaseChannelPlan(
+    test_stock_release_channel_plan.TestStockReleaseChannelPlan
```
Comment (simahawk) [inline stock_release_channel_plan_process_end_time/tests/test_stock_release_channel_plan.py]: hmm I'm not in favor of inheriting full test classes as every test you add there will run twice here.
Also overriding the method and calling super seems weird...
Can't you add a mixin and reuse it here?

**Comment by grindtildeath** [comment]
https://github.com/jbaudoux/wms/pull/2

**Comment by jbaudoux** [comment]
rebased & dropped do not merge commit

File: stock_release_channel_plan_process_end_time/tests/test_stock_release_channel_plan.py (hunk 1 above)
Comment (jbaudoux) [reply stock_release_channel_plan_process_end_time/tests/test_stock_release_channel_plan.py]: done

**Comment by jbaudoux** [comment]
Tests have been fixed. Can you update your review @lmignon @simahawk 

cc @sebalix

**Comment by lmignon** [comment]
@jbaudoux This one will collide with stock_release_channel_end_time. Once installed, it's no more possible to let the `process_end_date` be computed by default since it's a required field on the wizard. Moreover, the `process_end_date` is set by default to 'today'. As result when you launch the wizard with release channels configured with a `process_end_time` prior to the current time, the new computed `process_end_date` for the release channels will be in the past. What do you think about?

**Review by sbejaoui — APPROVED** [review]
code review + functional test

**Review by lmignon — APPROVED** [review]

**Pull request merged**